### PR TITLE
mod_acl_adminonly: Remove the module from Zotonic core

### PR DESCRIPTION
**Pull request by ddeboer**
Requires #1287, #1284.

Ref #1131.

### CODE REVIEW

**Comment by arjan** [comment]
I guess this is only for the 1.x release?
Also it might be good to add this to the upgrade notes, how to migrate from `mod_acl_adminonly` to `mod_acl_user_groups`.

**Comment by ddeboer** [comment]
Indeed, although we can introduce this change earlier, it’s probably easiest to do this for 1.0. Will add upgrade notes.

**Comment by ddeboer** [comment]
Added upgrade note.

File: doc/developer-guide/upgrading.rst [hunk 1]
```diff
@@ -7,6 +7,14 @@ These notes list the most important changes between Zotonic
...
 major Zotonic version.
 
+Upgrading from 0.x to 1.0
+-------------------------
+
+ACL
+"""
+
+* mod_acl_adminonly was replaced by :ref:`mod_acl_user_groups`. To create users
+  that have access to the admin, add them to the ‘Managers’ user group.
```
Comment (ddeboer) [inline doc/developer-guide/upgrading.rst]: This requires the default ACL rules for managers from #1284.

**Comment by mworrell** [comment]
👍 this module has its run and should indeed be shelved.

**Pull request merged**